components/layout.js

```diff
@@ -37,7 +38,7 @@ const Wrapper = glamorous.div((props, {fonts, colors}) => ({
   },
 }))
 
-function Layout({pathname, children, locale}) {
+function Layout({pathname, children, locale, content: {contributors}}) {
```

Comment:
Maybe instead of accepting this as a `content` prop, we should just accept a `contributors` prop. Then we could default it up here as well: `, contributors = []}`